apps/blemesh/syscfg.yml

```diff
@@ -26,8 +26,8 @@ syscfg.vals:
     # Default task settings
     OS_MAIN_STACK_SIZE: 768
 
-    # Newtmgr is not supported in this app, so disable newtmgr-over-shell.
-    SHELL_NEWTMGR: 0
+    # SMP is not supported in this app, so disable smp-over-shell.
```

Comment:
I'd emphasize here (and in other places too) that this is "Mcumgr SMP" and not "BLE SMP (Security Manager Protocol)"

Reply:
agree with @sjanc, also would be good to explain the abbreviation SMP in Mcumgr context

Reply:
Sounds good, I’ll add some more context